src/Sensors/IMU/IMU.cpp

```diff
@@ -1,51 +1,63 @@
 #include "IMU.h"
+#include <stdio.h>
```

Comment:
Do we need this include?

Reply:
Nope. I was print statement debugging and forgot to remove.